src/murmur/Messages.cpp

```diff
@@ -246,11 +246,7 @@ void Server::msgAuthenticate(ServerUser *uSource, MumbleProto::Authenticate &msg
 	}
 
 	Channel *lc;
-	if (bRememberChan) {
-		lc = qhChannels.value(readLastChannel(uSource->iId));
-	} else {
-		lc = qhChannels.value(iDefaultChan);
-	}
+	lc = qhChannels.value(readLastChannel(uSource->iId));
```

Comment:
This now disregards the `bRememberChannel` option

Reply:
I moved that option here:

https://github.com/deluxghost/mumble/blob/5f14a661ebb2bda5ba657c36da1082d4a98410b2/src/murmur/ServerDB.cpp#L1907

to make sure `rememberduration` is only enabled while `rememberchannel` is `true`

Reply:
Ah yes indeed. Must have missed it when skimming over the changes ^^